### ISSUE TITLE
poetry self:update on python 2.7 doesn't work

### ISSUE THREAD

**Issue by radix** [sdispater/poetry#53]
```
radix@pop-os:~$ poetry self:update -v
Updating to 0.8.4


  [RuntimeError]
  Unable to determine poetry's path


Exception trace:
 /home/radix/.local/lib/python2.7/site-packages/cleo/application.py in run() at line 94
   status_code = self.do_run(input_, output_)
 /home/radix/.local/lib/python2.7/site-packages/poetry/console/application.py in do_run() at line 74
   return super(Application, self).do_run(i, o)
 /home/radix/.local/lib/python2.7/site-packages/cleo/application.py in do_run() at line 197
   status_code = command.run(input_, output_)
 /home/radix/.local/lib/python2.7/site-packages/poetry/console/commands/command.py in run() at line 22
   return super(BaseCommand, self).run(i, o)
 /home/radix/.local/lib/python2.7/site-packages/cleo/commands/base_command.py in run() at line 146
   status_code = self.execute(input_, output_)
 /home/radix/.local/lib/python2.7/site-packages/cleo/commands/command.py in execute() at line 107
   return self.handle()
 /home/radix/.local/lib/python2.7/site-packages/poetry/console/commands/self/update.py in handle() at line 68
   self.update(release)
 /home/radix/.local/lib/python2.7/site-packages/poetry/console/commands/self/update.py in update() at line 102
   raise RuntimeError('Unable to determine poetry\'s path')

self:update [--preview] [--] [<version>]
```

**Comment by sdispater** [comment]
That's weird. It should not happen.

Could you execute:

```bash
python -c "import sys; print(sys.prefix, getattr(sys, 'base_prefix', None), getattr(sys,'real_prefix', None))"
```

**Comment by radix** [comment]
@sdispater  sorry I didn't get back to you sooner!

```
radix@pop-os:~$ python -c "import sys; print(sys.prefix, getattr(sys, 'base_prefix', None), getattr(sys,'real_prefix', None))"
('/usr', None, None)
```

**Comment by radix** [comment]
and it case it matters:

```
{}radix@pop-os:~$ python
Python 2.7.14 (default, Sep 23 2017, 22:06:14)
[GCC 7.2.0] on linux2
```

**Comment by radix** [comment]
```
{}radix@pop-os:~$ which poetry
/home/radix/.local/bin/poetry
```

**Comment by sdispater** [comment]
Ok, this is uncommon to me.

How did you install `poetry`? Since it is not under the prefix that your python knows about which leads to `poetry` not being able to find where it resides.

**Comment by radix** [comment]
I used `pip install --user`, which puts executables into `~/.local/bin`.

**Comment by radix** [comment]
just to add a little more context to this: ~/.local is used as the python root for `pip install --user` on linux. On OSX I believe it's a version-specific directory in `~/Library/Python/2.7` or something.

**Comment by sdispater** [comment]
Commit 5ac9794 on the `develop` branch will likely fix the issue. It will be available in the next `0.9.0` release.

**Comment by sdispater** [comment]
Release `0.9.0` is out!
